spec/integration/security_guides_spec.rb

```diff
@@ -0,0 +1,130 @@
+# frozen_string_literal: true
+
+require 'swagger_helper'
+
+describe 'Security Guides', swagger_doc: 'v2/openapi.json' do
+  before do
+    allow_any_instance_of(Authentication).to receive(:identity_header).and_return(
+      Insights::Api::Common::IdentityHeader.new(
+        Base64.strict_encode64(
+          {
+            'identity': {
+              'org_id': '1234',
+              'type': 'User',
+              'user': {
+                'email': 'a@b.com',
+                'username': 'a@b.com',
+                'first_name': 'a',
+                'last_name': 'b',
+                'is_active': true,
+                'locale': 'en_US'
+              },
+              'internal': {
+                'org_id': '29329'
+              }
+            },
+            'entitlements':
+              {
+                'insights': {
+                  'is_entitled': true
+                }
+              }
+          }.with_indifferent_access.to_json
+        )
+      )
+    )
+    stub_rbac_permissions(Rbac::COMPLIANCE_ADMIN, Rbac::INVENTORY_HOSTS_READ)
```

Comment:
I'd logically separate this line:
```suggestion
    )
    
    stub_rbac_permissions(Rbac::COMPLIANCE_ADMIN, Rbac::INVENTORY_HOSTS_READ)
```